frontend/src/routes/dashboard/admin/index.tsx

```diff
@@ -1,12 +1,110 @@
-import { component$ } from '@builder.io/qwik';
+import {
+  component$,
+  useResource$,
+  Resource,
+  useVisibleTask$,
+  useStore,
+  $,
+  PropFunction,
+} from '@builder.io/qwik';
 import type { DocumentHead } from '@builder.io/qwik-city';
+import { UserCtx } from '~/routes/layout';
+import { authorizedFetch } from '~/shared/auth.service';
+import type {
+  PaginatedRows,
+  PaginationParams,
+} from '~/components/table/server-paginated-data-table';
+import { ServerPaginatedDataTable } from '~/components/table/server-paginated-data-table';
+import { hybridPaginationHook, PaginationFetcher } from '~/hooks/hybridPaginationHook';
+
+export const serializeQueryUserPaginationParams = (paginationParams: PaginationParams) => {
+  const paramsForQuery: { [key: string]: string } = {
+    limit: '' + paginationParams.limit,
+    page: '' + paginationParams.page,
+  };
+  if (paginationParams.filter) {
+    paramsForQuery.filter = paginationParams.filter;
+  }
+  if (paginationParams.sortColumn && paginationParams.sort) {
+    paramsForQuery[`sort[${paginationParams.sortColumn}]`] = paginationParams.sort;
+  }
+
+  return new URLSearchParams(paramsForQuery).toString().replace(/%5B/g, '[').replace(/%5D/g, ']');
```

Comment:
Hey,

You can use URLSearchParams.
ref: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/append#examples